src/custom-element.js

```diff
@@ -793,10 +800,15 @@ export function createAmpElementProto(win, name, opt_implementationClass) {
       this.readyState = 'complete';
       this.layoutCount_++;
       this.toggleLoading_(false, /* cleanup */ true);
-      if (this.layoutCount_ == 1) {
```

Comment:
Here's what I understand, say if `layoutCallback` has loading error first time and success the second time. `this.layoutCount_` will equal to 2 here, and `firstLayoutCompleted()` will never be called.